zadani/soft/index.html

```diff
@@ -172,19 +175,18 @@ <h2>Fotogalerie</h2>
 
         </section><!-- konec .galerie -->
 
+    </dev>
+    <!-- patička -->
 
-
-        <!-- patička -->
-
+    <dev class="paticka">
         <p>
             Texty převzaty z článků:<br>
             Roman Pech - <a href="http://cestovani.idnes.cz/island-a-roman-pech-0nh-/kolem-sveta.aspx?c=A140519_125122_kolem-sveta_tom">Závislost jménem Island. Tipy znalce, jak zemi levně procestovat</a><br>
             Michal Balada - <a href="http://www.pohora.cz/tipy-na-vylety/16352-cesta-kolem-islandu-za-10-dni/">Cesta kolem Islandu za 10 dní</a>
         </p>
-        <p>
```

Comment:
Tohle tu mohlo zůstat, ale vidím, že v zadání to není, takže chápu, proč jsi smazala.